api/httpjson/auth.go

```diff
@@ -0,0 +1,66 @@
+package httpjson
+
+import (
+	"crypto/sha256"
+	"crypto/subtle"
+	"net/http"
+	"strings"
+)
+
+type RPCAuthType byte
+
+const (
+	UsernamePassword RPCAuthType = 0x01
+)
+
+type Autherization interface {
```

Comment:
Looks like typo. It should be 'Authorization'

Reply:
thanks!